src/SFA.DAS.Payments.Audit.RequiredPaymentService/Infrastructure/Ioc/AuditRequiredPaymentServiceModule.cs

```diff
@@ -40,12 +44,23 @@ protected override void Load(ContainerBuilder builder)
                 .As<IHandleMessageBatches<RequiredPaymentEventModel>>()
                 .InstancePerLifetimeScope();
 
+            builder.Register(c =>
```

Comment:
Seems like a lot of replicated code here, I think this registration could be added to the common audit autofac module in SFA.DAS.Payments.Audit.Application.Infrastructure.Ioc.AuditModule instead.